tests/AssertTest.php

```diff
@@ -717,7 +717,7 @@ public function getStringConversions()
      */
     public function testConvertValuesToStrings($method, $args, $exceptionMessage)
     {
-        $this->expectException('\InvalidArgumentException', $exceptionMessage);
+        $this->expectException('\InvalidArgumentException');
```

Comment:
Why was this changed? Is that syntax for old PHPUnt?

Reply:
Yes, this was the old syntax. The new one doesn't accept a second argument anymore. `$this->expectExceptionMessage` should be used as a replacement.